Sources/BuildSystemIntegration/ExternalBuildSystemAdapter.swift

```diff
@@ -178,6 +185,48 @@ actor ExternalBuildSystemAdapter {
     ).connection
   }
 
+  private func getConfigPath() -> AbsolutePath? {
+    var buildServerConfigLocations: [AbsolutePath?] = []
+
+    let workspacePath = projectRoot.appending(component: ".bsp")
+    buildServerConfigLocations.append(workspacePath)
+
+    #if os(Windows)
+    let localAppDataPath = AbsolutePath(validatingOrNil: ProcessInfo.processInfo.environment["LOCALAPPDATA"])
+    let programDataPath = AbsolutePath(validatingOrNil: ProcessInfo.processInfo.environment["PROGRAMDATA"])
+
```

Comment:
You need to search `%LOCALAPPDATA%\bsp\`, not `%LOCALAPPDATA%`. Same for `%PROGRAMDATA%`, `$XDG_DATA_HOME`, `$XDG_DATA_DIRS` and the macOS search locations.

Reply:
You need to add `bsp` to the path returned by the environment variable, not the environment variable’s key.